### ISSUE TITLE
add styling for Languages List

### ISSUE THREAD

**Issue by dwhieb** [digitallinguistics/app#168]
Wait to do this until after the `aria-describedby` attribute has been implemented (#167). The HTML for this component should be finalized then.
